_episodes/07-command_line.md

```diff
@@ -49,81 +48,172 @@ for num1 in range(0,num_atoms):
              bond_length_12 = calculate_distance(coord[num1], coord[num2])
              if bond_check(bond_length_12) is True:
                  print(F'{symbols[num1]} to {symbols[num2]} : {bond_length_12:.3f}')
-```
+~~~
 {: .language-python}
 
 Exit your text editor and go back to the command line.  Now all you have to do to run your code is type
 
-```
+~~~
 $ python geom_analysis.py
-```
+~~~
 {: .language-bash}
 in your Terminal window.  Your code should either print the output to the screen or write it to a file, depending on what you have it set up to do.  (The code example given prints to the screen.)
 
 ## Changing your code to accept user inputs
-In your current code, the name of the xyzfile to analyze, "water.xyz", is hardcoded; in order to change it, you have to open your code and change the name of the file that is read in.  If you were going to use this code to analyze geometries in your research, you would probably want to be able to specify the name of the input file when you run the code, so that you don't have to change it every single time.  These types of user inputs are called *arguments* and to make our code accept arguments, we have to import a new python library in our code.  
+In your current code, the name of the xyzfile to analyze, "water.xyz", is hardcoded; in order to change it, you have to open your code and change the name of the file that is read in.  If you were going to use this code to analyze geometries in your research, you would probably want to be able to specify the name of the input file when you run the code, so that you don't have to change it every single time. You might want to use the script like this:
+
+~~~
+$ python geom_analysis.py water.xyz
+~~~
+{: .language-bash}
+
+These types of user inputs are called *arguments* and to make our code accept arguments, we have to import a new python library in our code.  
+
 
 Open your geometry analysis code in your text editor and add this line at the top.
 
 ~~~
-import sys
+import argparse
 ~~~
 {: .language-python}
 
-Now that you have imported the `sys` library, you can use its functions.  The library has a function called `sys.argv()` which creates a list of all the arguments the user enters at the command line.  Everything after *python* is an argument, so `sys.argv[0]` is always the name of your script.  We would like our code to accept the name of the xyz file we want to analyze as an argument.  Add this line to your code.
-```
-xyzfilename = sys.argv[1]
-```
+We are importing a library called [https://docs.python.org/3/library/argparse.html](argparse) which can be used to easily make scripts with command line arguments. `Argparse` has the ability to allow us to easily write documentation for our scripts as well.
+
+First, we have to tell `argparse` what are arguments are. We tell argparse that we want to add some arguments using the following command:
```

Comment:
I think you mean "what our arguments are"

Reply:
I would remove the sentence "First, we have to tell.... arguments are".  Because the first thing you are doing is initializing argparse, and then later you are going to tell it what the arguments.  I would say something more like, first we initialize argparser by telling it that we want to add some arguments using the following command:

Reply:
> I think you mean "what our arguments are"

Definitely.

Reply:
> I would remove the sentence "First, we have to tell.... arguments are". Because the first thing you are doing is initializing argparse, and then later you are going to tell it what the arguments. I would say something more like, first we initialize argparser by telling it that we want to add some arguments using the following command:

Do you think we should use the word "initialize"?